PROJECT

```diff
@@ -2,6 +2,6 @@ domain: dell.com
 repo: github.com/dell/csi-baremetal-operator
 resources:
 - group: csi-baremetal
-  kind: Deployment
```

Comment:
Please leave it as `deployment`. Name `csibaremetals.csi-baremetal.dell.com` very confusing.

Reply:
Fixed